polish-adblock-filters/adblock.txt

```diff
@@ -3,20 +3,22 @@
 ! Oficjalne polskie filtry do AdBlocka, uBlocka Origin i AdGuarda
 ! Collaborators: Dominik, Wiesiek, Robert, Marek, buuuuuuu190, Caleb, Ryszard, Kruk, Weronika, Adam, xxcriticxx, maxoku, F4z, mathew85, Piter432, Adam K, Piotr S, Szymon D, Marek R, adreporter, kacper93, FadeMind, kerbenii, gzenio22, blocker999, FX54, hawkeye116477, gwarser, KonoromiHimaries
 ! Homepage: https://www.certyficate.it
-! Uzywanie polskich filtrow jest jednoznaczne ze świadomością co do ich zawartosci (co blokują itp.) a twórcy nie ponoszą odpowiedzialności za niewiedze uzytkownika
+! Używanie polskich filtrow jest jednoznaczne ze świadomością co do ich zawartosci (co blokują itp.) a twórcy nie ponoszą odpowiedzialności za niewiedze użytkownika.
 ! Works best with EasyList - be sure to subscribe it as well!
+! Działa najlepiej z EasyList - pamiętaj aby zasubskrybować!
 ! Last modified: 20 January 2018
 ! Expires: 1 day
-! Version: 2018012006
-! Support: 
+! Version: 2018012101
+! Zgłoszenia: 
 !   Email >> errors@certyficate.it
-!   Discord >> http://ds.certyficate.it
+!   Discord >> https://discord.me/polskiefiltry
 !   Github >> https://github.com/MajkiIT/polish-ads-filter/issues
-! License: https://creativecommons.org/licenses/by-nc-sa/4.0/
+! Wsparcie: https://patronite.pl/polskiefiltry
```

Comment:
Zamiast „wsparcie" powinno być raczej „dotacje".

Reply:
niekoniecznie. patronite to platforma która pozwala zostawić subskrybcje, by comiesięczne przekazywać określoną sumę pieniędzy.

wsparcie kojarzy się bardziej z ciągłością, z realnym wpływem na rozwój projektu.
a dotacja z jednorazową darowizną o nie określonym nominale.

Reply:
Ale support czy tam wsparcie to raczej może kojarzyć się ze zgłaszaniem błędów.

Reply:
sądzę że "patronite.pl" w linku wystarczy. nie uważam za sensowne rozpisywać się na ten temat w opisie filtra do adblocka : D

angielski różni się od polskiego i nie należy zakładać że oba języki są takie same.